src/App.js

```diff
@@ -3,35 +3,37 @@ import './App.css';
 import Header from './Header.js';
 import PokeList from './PokeList.js';
 import pokeData from './data.js';
-// import SearchBar from './SearchBar.js';
+import SearchBar from './SearchBar.js';
 
 
-// 1) move filter state to App.js
+// 1) move input state to App.js
 // 2) give SearchBar the handleChange to change App.js state
 // 3) give pokeList the filter (since it's the component that needs it)
 export default class App extends React.Component {
   state = {
-    input: ''
-    // filter: ''
+    input: '',
+    filter: ''
   }
+
   handleChange = e => {//tracks the state of the input
```

Comment:
as a hint, your handleChange and handleClick are duplicating each-other, but can be used to compare your input and filter state together onclick of your button